src/test/index.test.js

```diff
@@ -1,6 +1,6 @@
-let FS = require('fs');
+import * as FS from 'fs';
```

Comment:
You may directly import the method you will use like `{readFileSync}`

Reply:
it's done